ckanext/ontario_theme/resource_upload.py

```diff
@@ -168,5 +168,6 @@ def _check_zip_mimetype(self):
         if url and not (resource.get('url_type') == 'upload') and not resource.get('format'):
             resource['format'] = 'WEB'
 
-        if not (resource.get('format') == 'GeoJSON'):
-            resource['format'] = resource['format'].upper()
\ No newline at end of file
+        if resource.get('format') and not (resource.get('format') == 'GeoJSON'):
+            resource_format = resource.get('format')
+            resource['format'] = resource_format.upper()
```

Comment:
why not use 
```resource['format'] = resource.get('format').upper()```

 if `resource_format` is not being used elsewhere?

Reply:
Done